7.1 Snapshot.md

```diff
@@ -0,0 +1,53 @@
+# 7.1 Snapshot
+
+## 7.1.1 Introduction
+This section defines the *Snapshot* protocol.
+
+
+## 7.1.2 Dependencies
+The *Snapshotting* protocol depends on:
+
++ Timestamps: Core module to enable pruning.
++ Tip Selection Algorithm: No messages can be referenced for the part of the Tangle that is being pruned.
++ UTXO: Transactions whose outputs are unspent, remain after the snapshot.
++ Branches: We require that the at the point of pruning only master branch and rejected branch exist.
++ Markers: We require that the part of the Tangle being pruned is not actively dependent on Markers any more. 
++ Approval weight: We require that the approval weight is concluded at a time much before messages reach the pruning age.
++ Voting: We require that any voting is concluded much before messages reach the pruning age. 
++ Data flow: A weak past cone check is applied.
+
+
+## 7.1.3 Proof of Inclusion and Pruning
+
+In order to keep the ledger concise, data that is sufficiently old can be removed. We call this process pruning. Thus, the IOTA 2.0 protocol for a standard node does not attempt to keep all information back until the Genesis. Rather the protocol effectively becomes a mechanism to prove whether a certain message was, at some point in the past, included in the ledger. This can be done e.g. through a Merkle tree proof. 
+
+Transactions are payloads of value messages, that cannot be pruned. Specifically when a transaction contains an unspent output, see [5.1. UTXO](./5.1%20UTXO.md), the transaction shall be kept when pruning the ledger. An optimization of this process is possible by reducing the message to only keeping the information about the unspent outputs, however the implementation will be more complicated, and thus is not considered for this first iteration of the protocol.
```

Comment:
This still says that we cannot prune value messages, but isnt the correct to say that we cannot rune them unless all their outputs are spent?

Reply:
you are correct. i rewrote it.